src/app/pages/common/ix-forms/components/ix-file-upload/ix-file-upload.component.html

```diff
@@ -0,0 +1,70 @@
+<div *ngIf="label" class="label-container">
+  <label class="label">
+    {{label}}
+    <span *ngIf="required" class="required">*</span>
+  </label>
+  <tooltip *ngIf="tooltip" [header]="label" class="tooltip" [message]="tooltip"></tooltip>
+</div>
+
+<div class="input-container" [class.disabled]="isDisabled">
+  <div>
+    <button
+      type="button"
+      mat-button
+      color="accent"
+      class="btn-default"
+      [disabled]="isDisabled"
+      (click)="fileBtnClick()"
+    >
+      {{ 'Choose File' | translate }}
+    </button>
+  </div>
+  <div *ngIf="fileList && fileList.length">
+    <div *ngFor="let file of fileList" class="mt-1">
+      {{ file.name }}
+    </div>
+  </div>
+  <div *ngIf="hideButton; else showButton" style="display: none;">
+    <input
+      *ngIf="!multiple; else multipleFiles"
+      id="fb"
+      type="file"
+      #fileInput
+      accept="{{ acceptedFiles }}"
+      [required]="required"
+      [disabled]="isDisabled"
+      (change)="upload(fileLocation)"
+    />
+    <ng-template #multipleFiles>
+      <input
+        id="fb"
+        type="file"
+        #fileInput
+        accept="{{ acceptedFiles }}"
+        [required]="required"
+        [disabled]="isDisabled"
+        (change)="upload(fileLocation)"
+        multiple
```

Comment:
This looks the same as another input with only difference being multiple attribute. Can you use [attr.multiple] instead somehow in one tag?